gnc/sub8_perception/nodes/follow_orange_pipes.py

```diff
@@ -54,6 +54,11 @@ def get_ncc_scale(self):
 
         a = np.array(self.occ_grid.cam.project3dToPixel([0.0, self.channel_width, height]))
         b = self.occ_grid.cam.project3dToPixel([0.0, 0.0, height])
+        try:
+            int(np.linalg.norm(a - b) / 2.0)
+        except:
```

Comment:
Random bugs when doing simulation testing, not sure the cause...
It would throw an error saying it was nan